ui/src/app/images/views/ImageList/UbuntuImages/DefaultSource/DefaultSource.tsx

```diff
@@ -32,7 +35,11 @@ const DefaultSource = (): JSX.Element => {
   const ubuntu = useSelector(bootResourceSelectors.ubuntu);
   const resources = useSelector(bootResourceSelectors.ubuntuResources);
   const saving = useSelector(bootResourceSelectors.savingUbuntu);
+  const previousSaving = usePrevious(saving);
+  const error = useSelector(bootResourceSelectors.eventErrors)[0]?.error;
```

Comment:
I think this needs to look for an event with type `BootResourceAction.STOP_IMPORT` doesn't it?